### DISCUSSION TITLE
v1.6.0-preview2

### DISCUSSION THREAD

**Discussion by duncanmacmichael** [microsoft/WindowsAppSDK#4676] [Announcements]
## 1.6 Preview 2 (1.6.240821007-preview2) 🔃

The second preview of WinAppSDK 1.6 is now available! Check out the [release notes](https://aka.ms/windowsappsdk/1.6/1.6-preview2/release-notes) for more information on the new & updated features and known issues.

### Updating to this version
The 1.6-preview2 package is available at https://www.nuget.org/packages/Microsoft.WindowsAppSDK/1.6.240821007-preview2

### Providing feedback
As we introduce new & upcoming features in our preview releases, we appreciate all of your feedback as we work towards a stable release:
- For WinUI 3 feedback, please open an issue on the [WinUI repo](https://github.com/microsoft/microsoft-ui-xaml/issues/new/choose).
- For feedback regarding all other Windows App SDK components, please open an issue here, on the [Windows App SDK repo](https://github.com/microsoft/WindowsAppSDK/issues/new/choose).

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.6.0-preview2'>v1.6.0-preview2</a>.</em>

**Comment by ghost1372** [comment]
where is TitleBar control? why not included? can we see it in next release? or can we see it in v1.6?

**Reply by Zakariathr22** [reply]
Also waiting for it, I think it is one of the most necessary controls that should be included, hope we get it in the 1.6 version

**Reply by ghost1372** [reply]
I am aware that it was [announced](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/preview-channel#new-features-1) in the previous version that it is not currently available in non-experimental versions for some reason.
But I expected to see it in this version. The way it's going, I don't think we'll see it in version 1.6

**Reply by DarranRowe** [reply]
At this point it is pretty unlikely that the TitleBar control will be in 1.6. If it is, then expect a 1.6-preview 3 release.

**Reply by whiskhub** [reply]
They reluctantly announced it won't make it, at the last Windows Dev Chat: https://www.youtube.com/live/43rwZpcKtCA?t=3050
Not sure why they are always so secretive about things...

**Reply by ghost1372** [reply]
The important point is that we don't seem to see the title bar in 1.6, but we may see it in 1.6.x. It's good news that we don't have to wait for 1.7. (I think so because @michael-hawker  said that we don't see it in the initial version 1.6)
It is very unfortunate that everything fails and gets delayed. 
Bugs are not solved.
There is no news about the development status and everything is delayed, what a shame.
